pgraph/pgraph.go

```diff
@@ -387,11 +388,11 @@ func (g *Graph) Sprint() string {
 		str += fmt.Sprintf("Vertex: %s\n", v)
 	}
 	for _, v1 := range g.VerticesSorted() {
-		vs := []Vertex{}
+		var vs VertexSlice
```

Comment:
Please leave these alone for now. Explicitly seeing the list is more logical at least for me.